QQE-832.md

```diff
@@ -0,0 +1,39 @@
+# QQE-832 Coverage for Quarkus Cache with Infinispan
+
+JIRA link: https://issues.redhat.com/browse/QQE-832
+
+- Quarkus guide: https://quarkus.io/guides/cache-infinispan-reference
+
+## Scope of the testing
+- The test coverage will be verified for environments:  JVM, Native mode and Openshift.
+- Verify the `quarkus-infinispan-cache` extension using `@CacheResult`, `@CacheInvalidate`, `@CacheInvalidateAll` and `@CacheKey` caching annotations.
+- Cover different usages:
+  - from an application scoped service
+  - from a request scoped service
+  - from a blocking endpoint
+  - from a reactive endpoint
+- Use [custom POJOs](https://quarkus.io/guides/cache-infinispan-reference#marshalling-your-pojos) as cache value, verify marshalling in Quarkus Infinispan.
+- Test [cache expiration](https://quarkus.io/guides/cache-infinispan-reference#expiration) properties.
+
+## Getting familiar with the feature
+Following actions were taken to ensure familiarity:
+- Ensure documentation provides clear explanation on configuration options.
+- Ensure good user experience and simplicity of use.
+
+## Existing test coverage
+- [Quarkus integration tests infinispan-cache](https://github.com/quarkusio/quarkus/tree/main/integration-tests/infinispan-cache/src/test/java/io/quarkus/it/cache/infinispan)
+- [Quarkus extensions: infinispan-cache](https://github.com/quarkusio/quarkus/tree/main/extensions/infinispan-cache/deployment/src/test)
+
+### Impact on test suites and testing automation
+- New tests will be added to [cache](https://github.com/quarkus-qe/quarkus-test-suite/tree/main/cache) module in QQE TS.
+
+### Impact on resources
+- Tests in QE Test Suite increase execution time up to 1 minutes in JVM and 3 minutes in Native.
```

Comment:
And what about Openshift?

Also there seems to be a typo in "1 minutes".

Reply:
updated